server/services/database/users.js

```diff
@@ -0,0 +1,56 @@
+import db from "../../db";
```

Comment:
Its fine for a small project, but usually you should have any code which interacts with the DB, put in the "repository" layer 

so Controller -> Service -> Repository.